src/app/containers/header/header.component.html

```diff
@@ -5,7 +5,7 @@
       [navigation]="userNav"
       (navigate)="onNavigate($event)"
     ></app-hmcts-global-header>
-
+    <app-phase-banner type="beta"></app-phase-banner>
```

Comment:
why do we need type tag here?

Reply:
Hey Franco. This is a string value which is displayed in the component. The same in approve orgs.

Reply:
I am not sure how this is not erroring tbh shouldn't then this be [type]='beta'

Reply:
Hi Franko, In [type]='beta', beta is a variable and In type='beta' , beta is a string. these are angular data-binding  fundamentals